test/e2e/upgrades/services.go

```diff
@@ -73,7 +73,7 @@ func (t *ServiceUpgradeTest) Test(f *framework.Framework, done <-chan struct{},
 	case MasterUpgrade:
 		t.test(f, done, true)
 	default:
-		t.test(f, done, false)
+		t.test(f, done, true)
```

Comment:
This should not be the default for two reasons. 

  1. The enum could expand to upgrade types that you don't intend to be tested this way.
  2. Some cloud providers might not support a graceful node upgrade like this. There's should be a check for that.